File: Step1/Logical-Thinking/Pattern14/Solution.java
// SAMPLE PATTERN
// A 
// AB 
// ABC 
public class Main
{
    static void pattern(int n){
    for(int i = 1; i<=n; ++i){
        for(int j = 65; j<(65+i); ++j){
            System.out.print((char)j);
        }
        System.out.println();
    }
}
	public static void main(String[] args) {
		pattern(3);
	}
}

File: Step1/Logical-Thinking/Pattern14/Solution.cpp
#include <iostream>
using namespace std;
void pattern(int n){
    for(int i = 1; i<=n; ++i){
        for(int j = 65; j<(65+i); ++j){
            cout<<char(j);
        }
        cout<<endl;
    }
}
int main()
{
    pattern(3);

    return 0;
}
